create-snowpack-app/app-template-svelte-typescript/src/App.svelte

```diff
@@ -1,4 +1,4 @@
-<script>
```

Comment:
Doesn’t Svelte recommend `lang="ts"` instead?

Reply:
I put some thought into it. I went with typescript because for someone just starting out it takes fewer cycles to process than lang="ts". Svelte favors ts because it's shorter. I think templates should be more instructional than optimal, but I'm happy to change it to whatever you prefer.

Reply:
Thanks for the explanation! To be clear: I was asking as a Svelte novice, and genuinely didn’t know 🙂. I too favor the clarity of `typescript` now that you’ve explained it. Happy to merge.